src/Database/ModelInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php namespace Winter\Storm\Database;
+
+/**
+ * The Model interface defines required methods for models to work.
+ *
+ * At the moment, this only restricts the signature for the constructor to only allow attributes as a parameter, in
+ * order to allow several static calls to work within the model architecture, but still allow models to extend the
+ * constructor if they wish.
+ *
+ * @author Winter CMS
+ */
+interface ModelInterface
```

Comment:
It's in my notes in the PR, but basically, we are now enforcing a signature for the Model constructor that matches Laravel and prevent unsafe static calls. It's something that needs to be discussed, because it could *potentially* be a breaking change, but personally, I don't see a need for extra parameters in the constructor.